css/css-syntax-3/tokenizing-and-parsing/input-byte-stream/i18n-css-syntax-bytes-018.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html  lang="en" >
+<head>
+<meta charset="UTF-8"/>
+<title>CSS3 Syntax, input byte stream: css bom and @charset iso15</title>
+<link rel='author' title='Richard Ishida' href='mailto:ishida@w3.org'>
+<link rel='help' href='http://www.w3.org/TR/css3-syntax/#input-byte-stream'>
+<script src="../../../resources/testharness.js"></script>
+<script src="../../../resources/testharnessreport.js"></script>
+<link rel="stylesheet" type="text/css" href="../../resources/../testharness.css">
+<meta name='flags' content='http dom'>
+<meta name="assert" content="When a stylesheet has a BOM and a @charset declaration that is not for a Unicode encoding, the BOM takes precedence.">
+<style type='text/css'>
+.test div { width:50px; }
+
+p { color: gray; margin-top: 3em; font-size: 80%; }
+</style>
+<link rel="stylesheet" type="text/css" href="support/bom-charset15.css">
```

Comment:
This stylesheet has a `Content-Type` of `text/html`. Per <https://drafts.csswg.org/css-cascade/#content-type>, this should cause it to be ignored, since this is a standards-mode document.  Thus this test isn't testing quite the right thing.